x/ethbridge/keeper/keeper.go

```diff
@@ -190,15 +192,15 @@ func (k Keeper) ProcessUpdateCethReceiverAccount(ctx sdk.Context, cosmosSender s
 }
 
 // ProcessRescueCeth transfer ceth from ethbridge module to an account
-func (k Keeper) ProcessRescueCeth(ctx sdk.Context, msg types.MsgRescueCeth) error {
+func (k Keeper) ProcessRescueCeth(ctx sdk.Context, msg *types.MsgRescueCeth) error {
 	logger := k.Logger(ctx)
-	if !k.oracleKeeper.IsAdminAccount(ctx, msg.CosmosSender) {
+	if !k.oracleKeeper.IsAdminAccount(ctx, sdk.AccAddress(msg.CosmosSender)) {
```

Comment:
sdk.AccAddressFromBech32(msg.CosmosSender)
```suggestion
addAddr, err sdk.AccAddressFromBech32(msg.CosmosSender)
if err != nil {
return err 
}

	if !k.oracleKeeper.IsAdminAccount(ctx, accAddr) {
```

This should be called in order to check validity of the address